### ISSUE TITLE
Use default CMake find module for GLUT and FreeGLUT

### ISSUE THREAD

**Issue by dtorban** [ivlab/MinVR2#54]
I think the Default CMake find module for GLUT also includes FreeGLUT.  Perhaps we should just use the following instead of writing our own:

[How to use OpenGL/ Freeglut and CMake](http://igorbarbosa.com/articles/how-to-use-opengl-freeglut-and-cmake/)

At least I felt like I have done this in the past and it worked.

**Comment by dtorban** [comment]
It looks like it looks for both GLUT and FreeGLUT here:

[https://github.com/Kitware/CMake/blob/master/Modules/FindGLUT.cmake](https://github.com/Kitware/CMake/blob/master/Modules/FindGLUT.cmake)

**Comment by BenKnorlein** [comment]
Yeah. We should use that one. However the plugin, requires freeglut and will not work with glut, as glut will be in charge of the mainloop. Freeglut does not. So we would have to still run an additional check.
